server/commands/admin.py

```diff
@@ -128,8 +129,9 @@ def kickban(client, arg, ban_hdid):
             ban_id = int(args[1])
             unban_date = None
         except ValueError:
-            reason = args[1]
-            unban_date = arrow.get().shift(hours=6).datetime
+            reason = args[1]                     
+            bandur = client.server.config['bandur']
```

Comment:
This will cause an error if the configuration entry is missing in the YAML. You will need to use a try/except to cover this case.